packages/golden_toolkit/lib/src/testing_tools.dart

```diff
@@ -216,7 +220,29 @@ Future<void> screenMatchesGolden(
   }
 }
 
-// Matches Golden file is the easiest way for the images to be requested.
-Future<void> _primeImages(String fileName, Finder finder) => matchesGoldenFile(fileName).matchAsync(finder);
+/// A function that primes all assets by calling [matchesGoldenFile] with the tester.
+/// Doing so may have some unwanted side effects like the creation of a failures folder even when the test
+/// succeeds.
+///
+/// See also:
+/// * [GoldenToolkitConfiguration.primeAssets] to configure a global asset prime function.
+Future<void> defaultPrimeAssets(WidgetTester tester, String name, Finder finder) =>
```

Comment:
super minor, but can we name this legacyPrimeAssets?

Reply:
Done